igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisService.groovy

```diff
@@ -86,7 +95,7 @@ class TravisService implements BuildService {
 
     Build getBuild(String repoSlug, int buildNumber) {
         Builds builds = getBuilds(repoSlug, buildNumber)
-        return builds.builds.first()
+        return builds.builds.size() > 0 ? builds.builds.first() : null
```

Comment:
@lwander I did some tests against the travis api, changing build number to one that was to high and job name to a job that does not exist. Travis returns `{"builds":[],"commits":[]}` for both. So I think the answer is that it can't be null.

I added a test with `{"builds":[],"commits":[]}` as return value, just to have it in the tests.